### PR TITLE
Fixed functionality to mark an application on hold and send an email to the candidate

### PR DESCRIPTION
**Pull request by Bahugunajii**
Targets #1542 

### Description
- A "put on hold" button has been added to the edit application page.
- Width of the dropdown has been adjusted
- On putting the application "on hold", the user is redirected back to the open application page and the application is moved to the on-hold page with an "On hold" label status
- Also the applicant is sent an email that his/her application has been put on Hold

### Checklist
- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]
@Bahugunajii your PR has missing description and checklist section. Please add that so that I can move forward with the review

**Comment by P4NK4J** [comment]
@rathorevaibhav This PR is lacking some functionalities that need to be implemented. @Bahugunajii Please remove the ready to review label if you are currently working on this.

File: Modules/HR/Entities/ApplicationRound.php [hunk 1]
```diff
@@ -147,6 +149,31 @@ public function _update($attr)
...
+                $application->untag('in-progress');
+                $fillable['round_status'] = 'on-hold';
+                $application->onHold();
+
+                ApplicationMeta::create([
+                'hr_application_id' => $application->id,
+                'key' => 'on-hold',
+                'value' => json_encode([
+                'on-hold_by' => $fillable['conducted_person_id']])
+                ]);
```
Comment (kuldeep3) [inline Modules/HR/Entities/ApplicationRound.php:163]: indentation

**Review by kuldeep3 — APPROVED** [review]
Looks good one feedback

File: Modules/HR/Entities/ApplicationRound.php (hunk 1 above)
Comment (Bahugunajii) [reply Modules/HR/Entities/ApplicationRound.php]: > indentation
Corrected!

**Review by kuldeep3 — APPROVED** [review]

**Review by rathorevaibhav — APPROVED** [review]
Approving as @kuldeep3 already approved.

**Pull request merged**